### ISSUE TITLE
New Status needed "wasRetried"

### ISSUE THREAD

**Issue by gorbysbm** [extent-framework/extentreports-java#151]
Hello,
In TestNG version 7.1.0 there is a new test result status "wasRetried." Currently Extent Reports doesn't provide this test result status. Please add it.

**Comment by gorbysbm** [comment]
Just to be elaborate the purpose of this new status... if the test's final status after retries is "PASSED + wasRetried" then it should not show up as a Failure or as a Skip in the final test results.

**Comment by anshooarora** [comment]
Noted. All previously executed tests with the same name will be removed, not without taking into account their hierarchy.

**Comment by anshooarora** [comment]
Can you share a small sample that I can use? Possibly a Github repo?

**Comment by gorbysbm** [comment]
it's really straightforward. Lmk if you need more elaboration than what I'm pasting below:

```
	public void afterMethod(ITestResult result) throws Exception {
		//Since extent Reports doesn't yet have a "was retried" status remove the test to avoid false failure reports
		if (result.wasRetried()){
			HtmlReporter.removeCurrentNode();
		}
}
```

**Comment by gorbysbm** [comment]
> Noted. All previously executed tests with the same name will be removed, not without taking into account their hierarchy.

Yes, we would remove the failed test, but we would +1 the wasRetried count for every retry.

**Comment by anshooarora** [comment]
Thanks @gorbysbm

**Comment by anshooarora** [comment]
Support is available now,  you can test with testng-adapter `1.2.0-SNAPSHOT` with `ITestListener`.   

Here are the updates: https://github.com/extent-framework/extentreports-testng-adapter/tree/v1.2.x

**Comment by anshooarora** [comment]
This is available starting testng plugin `1.2.3`.
